simple_salesforce/api.py

```diff
@@ -411,6 +411,12 @@ def __init__(self, object_name, session_id, sf_instance, sf_version='27.0', prox
         self.request = requests.Session()
         self.request.proxies = proxies
 
+        self.headers = headers if headers else {
```

Comment:
I thought about this, but I opted to have the user define it rather than instituting a merge policy when the same key is redefined/changed in the new header passed in. Probably a little too prudent, and I'd be fine with overwriting the new value for a given key if it's defined in the headers passed in. Thoughts?